dependencies/seccomp/meson.build

```diff
@@ -0,0 +1,23 @@
+# OS Specific checks 
+#============================================================================
+
+
+
+# Look for libsecppomp, required for Linux sandboxing.
+#----------------------------------------------------
+if sys_name.contains('linux')
```

Comment:
you don't need contains here unless you're planning to pass something other than straight `'linux'` in a cross file, we always return just `'linux'`, not `'linux<number>'`, so `sys_name == 'linux'` is both safe and idiomatic.